### PR TITLE
Final

### PR DESCRIPTION
**Pull request by kai9987kai**
- Fixed  modern scroll bar
- Custom Cursor
- Disable right click
- Save scroll location to javascript local storage
- Scroll to the top button

### CODE REVIEW

**Comment by djpiper28** [comment]
Does not compile. Try using `npm i && npm run dev` and fixing this issue.

```
[plugin:vite:react-babel] /home/danny/github/my-website/src/main.jsx: Unexpected token (53:2)
  56 | }

/home/danny/github/my-website/src/main.jsx:53:2

51 |   <style> element and add custom cursor style
52 |  function setCursorColor() {
53 |    const styleElement = document.createElement('style');
   |    ^
54 |    styleElement.innerHTML = 'html { cursor: url("https://raw.githubusercontent.com/kai9987kai/kai9987kai.github.io/master/cur.png"), auto; }';
55 |    document.head.appendChild(styleElement);

    at Parser._raise (/home/danny/github/my-website/node_modules/@babel/parser/lib/index.js:476:17)
    at Parser.raiseWithData (/home/danny/github/my-website/node_modules/@babel/parser/lib/index.js:469:17)
    at Parser.raise (/home/danny/github/my-website/node_modules/@babel/parser/lib/index.js:430:17)
    at Parser.unexpected (/home/danny/github/my-website/node_modules/@babel/parser/lib/index.js:3789:16)
    at Parser.parseExprAtom (/home/danny/github/my-website/node_modules/@babel/parser/lib/index.js:12622:22)
    at Parser.parseExprAtom (/home/danny/github/my-website/node_modules/@babel/parser/lib/index.js:7812:20)
    at Parser.parseExprSubscripts (/home/danny/github/my-website/node_modules/@babel/parser/lib/index.js:12149:23)
    at Parser.parseUpdate (/home/danny/github/my-website/node_modules/@babel/parser/lib/index.js:12129:21)
    at Parser.parseMaybeUnary (/home/danny/github/my-website/node_modules/@babel/parser/lib/index.js:12104:23)
    at Parser.parseMaybeUnaryOrPrivate (/home/danny/github/my-website/node_modules/@babel/parser/lib/index.js:11901:61

Click outside or fix the code to dismiss.
You can also disable this overlay by setting server.hmr.overlay to false in vite.config.js.
```

File: src/main.js
Comment (djpiper28) [inline src/main.js:1]: this code can be moved to App.jsx surely

**Review by djpiper28 — CHANGES_REQUESTED** [review]

File: src/main.js
Comment (kai9987kai) [reply src/main.js:1]: js and jsx seem to not work together

File: src/main.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  }
+}
+
+// Listen for scroll event and save scroll position
+window.addEventListener('scroll', saveScrollPosition);
+
+// Load scroll position on page load
+window.addEventListener('load', loadScrollPosition);
+
+ <style> element and add custom cursor style
```
[the remark this answers is not in the document]
Comment (kai9987kai) [reply src/main.js]: do you know the error message I don't have react installed or anything

**Comment by kai9987kai** [comment]
might have fixed something

**Comment by djpiper28** [comment]
Install npm in wsl or something then run npm I then npm run Dev. I would fix it but I'm on the piss tonight.

**Pull request closed without merging**